types/model/name.go

```diff
@@ -25,11 +26,17 @@ var (
 
 // Defaults
 const (
-	// DefaultMask is the default mask used by [Name.DisplayShortest].
-	DefaultMask = "registry.ollama.ai/library/_:latest"
+	// MaskDefault is the default mask used by [Name.DisplayShortest].
+	MaskDefault = "registry.ollama.ai/library/?:latest"
```

Comment:
`DefaultMask` reads nicer IMO. Similarly, `NothingMask`. Though a common prefix is nice

Reply:
I agree but I was going for symmetry. I'd like to keep it prefixed with "kind" for now. Because it is the default, it never needs to be typed outside this package.